frontend/src/components/QuestionPane.jsx

```diff
@@ -34,44 +38,86 @@ const parse = (text) => {
 }
 
 const QuestionPane = ({ id }) => {
-  const { data, loading } = useFetchQuestionById(id)
+  const [questionId, setQuestionId] = useState(id)
+  const [questionData, setQuestionData] = useState([])
+
+  useEffect(() => {
+    const getQuestion = async () => {
+      await axios
+        .get(URL_QUESTION_SERVICE + '?id=' + questionId)
+        .then((response) => {
+          var newData = response.data
```

Comment:
```suggestion
          const newData = response.data
```

https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994